jspwiki-event/pom.xml

```diff
@@ -62,5 +62,10 @@
       <artifactId>junit-jupiter-engine</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.jspwiki</groupId>
+      <artifactId>jspwiki-util</artifactId>
```

Comment:
needs `<version>${project.version}</version>` and `groupId` can be replaced with `${project.groupId}` (see last comment on main pom.xml file)